src/api/users/users.router.js

```diff
@@ -0,0 +1,8 @@
+import Router from 'express';
+import * as usersController from './users.controller.js';
+
+const router = Router();
+
+router.get('/id/:id', usersController.getById);
```

Comment:
In the **gameTitle.getById** route, the current implementation is:

`router.get('/:id', gameTitlesController.getById);`

We should aim to maintain a consistent approach for all routes.